.env.template

```diff
@@ -0,0 +1,2 @@
+# If testing against a local version of the API, you can override this.
```

Comment:
_Random place for comment_

Prob a user error but getting `No deployed code found for contract`

```
Repo URL
**https://github.com/garyghayrat/safe-send-contracts**
Commit Hash
**242f51322afd273b3ad284d54b97ae25e73dd8b2**
Contract Address
**0xBFA223E28B3B9d0B344eBE3FCB976Bdd173ACCaE**
Build Configuration
**Foundry - default**
Creation Transaction Hashes
**Sepolia - 0x5f71b652746e93982eb5f74d1e9b88b29a90b5bf4273f4e50000967ae4131a5d**
```

Reply:
Have you tried this on the deploy preview link? (It's linked in the github actions status below: https://deploy-preview-7--cove-frontend-0.netlify.app/)

Reason being you may just be missing env vars or have an incorrect config, but jumping straight to the deploy preview can help figure out if it's a config issue or actual issue

Reply:
Just tried the deploy preview and seeing the same error

Reply:
Just pushed a fix, thanks! https://github.com/ScopeLift/cove-backend/commit/8f5cdd81273f09c1d73cf4ed89959b467f6fa659

If you run the backend locally it should work now, new version should be deployed to the server shortly

Reply:
Working great now, thanks!